docs/Getting Started/Install and Run/index.md

```diff
@@ -76,7 +77,7 @@ From there, you can build and install the project, run the following commands in
 
 ```bash
 make install-3rd_party
-make install
+make install -j3
```

Comment:
```suggestion
make install
```
Probably best to leave this as-is. I'll rewrite things to hopefully be clearer in another PR